pinot-controller/src/main/java/com/linkedin/pinot/controller/validation/ValidationManager.java

```diff
@@ -140,7 +140,14 @@ public void runValidation() {
       List<SegmentMetadata> segmentMetadataList = new ArrayList<SegmentMetadata>();
 
       TableType tableType = TableNameBuilder.getTableTypeFromTableName(tableName);
-      TableConfig tableConfig = null;
+      TableConfig tableConfig = _pinotHelixResourceManager.getTableConfig(tableName);
+      // Validate that the table config is in proper JSON format
+      try {
+        tableConfig.toJSONConfigString();
+      } catch (Exception e) {
+        LOGGER.error("Invalid JSON table config for table {}", tableName);
```

Comment:
Do u want to continue to the next table if the table config or schema is invalid?